angular-ngrx-scss/src/app/home/profile/profile-nav/profile-nav.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ProfileState } from 'src/app/state/profile/profile.state';
+
+@Component({
+  selector: 'app-profile-nav',
+  templateUrl: './profile-nav.component.html',
+  styleUrls: ['./profile-nav.component.scss'],
+})
+export class ProfileNavComponent {
+  @Input()
+  profile$?: Observable<ProfileState>;
```

Comment:
If my last comment is correct then username will be needed. We should discuss this.

Reply:
Yeah let's get the folder moved up a level and then see what we need. I think if we do keep the username (which is potentially fine) we should adjust this to pass just the username as the `Input`, instead of the full profile (since username is all we're using from the profile here).